conf/crg.config

```diff
@@ -5,11 +5,30 @@ params {
     config_profile_url = 'http://www.linux.crg.es/index.php/Main_Page'
 }
 
-process {
-    executor = 'crg'
-    queue    = 'short-centos79,long-centos79'
+def host = System.getenv("HOSTNAME")
+
+if (host ==~ /^genoa64.*/) {
+    process {
+        executor = 'slurm'
+        queue    = 'genoa64'
+        clusterOptions = {
+            task.time <= 1.h    ? '--qos=vshort' :
+            (task.time <= 3.h   ? '--qos=shorter' :
+            (task.time <= 6.h   ? '--qos=short' :
+            (task.time <= 12.h  ? '--qos=normal' :
+            (task.time <= 24.h  ? '--qos=long' :
+            (task.time <= 48.h  ? '--qos=vlong' :
+            (task.time <= 168.h ? '--qos=marathon' :'--qos=ethernal' )))))) }
+    }
+} else {
+    process {
+        executor = 'crg'
```

Comment:
What is this `crg` executor?

Reply:
Is a variation of the `sge` executor that @pditommaso implemented for the institute but it is not documented.